README.md

```diff
@@ -674,29 +673,29 @@ createConnection(/*...*/).then(async connection => {
     metadata.orientation = "portait";
     metadata.photo = photo; // this way we connect them
 
-    // get entity repositories
+    // Get entity repositories
     let photoRepository = connection.getRepository(Photo);
     let metadataRepository = connection.getRepository(PhotoMetadata);
 
-    // first we should persist a photo
+    // First we should persist a photo
     await photoRepository.persist(photo);
 
-    // photo is saved. Now we need to persist a photo metadata
+    // Photo is saved. Now we need to persist a photo metadata
     await metadataRepository.persist(metadata);
 
-    // done
-    console.log("metadata is saved, and relation between metadata and photo is created in the database too");
+    // Done

```

Comment:
for code comments its quite common convention to start code block comment with a small latter